version.txt

```diff
@@ -1 +1 @@
-10.0.1-SNAPSHOT
+10.1.0-SNAPSHOT
```

Comment:
We only release patch versions directly from `releases`, so `10.0.1` was correct

Reply:
I don't recall having updated this... 🤷